packages/sn-client-utils/src/index.ts

```diff
@@ -10,3 +10,5 @@ export { sleepAsync } from './SleepAsync'
 export { Trace } from './Trace'
 export { toNumber } from './ToNumber'
 export { tuple } from './Tuple'
+export * from './AsyncReturnValue'
+export * from './PromiseReturnValue'
```

Comment:
I'm not sure if these types are needed.

Reply:
There are only one usage here: https://github.com/SenseNet/sn-client/pull/393/files#diff-240e5c83bc40b7b5b3fe234adbd6d05eR59
We can remove them, but then we have to use an inline type / explicit type here.